stablehlo/reference/Element.h

```diff
@@ -149,18 +155,10 @@ Element atan2(const Element &e1, const Element &e2);
 /// individually equal modulo the tolerance.
 Element areApproximatelyEqual(const Element &e1, const Element &e2);
 
-/// Returns bitcast converted Element(s). Let E and E' be the element types of
-/// `elements[0].getType()` and `type` respectively and member of `elements` has
-/// the same type:
-// * size(elements) = 1 and num_bits(E') < num_bits(E):
-//   returns n Elements such that n = bitwidth(E)/bitwidth(E') and n *
-//   num_bits(E') == num_bits(E).
-// * size(elements) > 1 and num_bits(E') > num_bits(E):
-//   returns a single Element such that size(elements) * num_bits(E) ==
-//   num_bits(E').
-// * size(elements) = 1 and num_bits(E') == num_bits(E):
-//   returns a single Element of type `type`.
-SmallVector<Element> bitcastConvert(Type type, ArrayRef<Element> elements);
+/// Various flavors of bitcast conversion as defined in the specification.
+Element bitcastConvertOneToOne(Type type, const Element &e);
+SmallVector<Element> bitcastConvertOneToMany(Type type, const Element &e);
+Element bitcastConvertManyToOne(Type type, ArrayRef<Element> es);
```

Comment:
Also, I added checks at the beginning of these functions to make sure that it's indeed one-to-one, one-to-many or many-to-one (i.e. that the relevant bitwidths are multiples of each other, etc), and report_fatal_error if checks fail. This is tricky stuff, so it's best to be sure.